tests/kernel/gen_isr_table/testcase.yaml

```diff
@@ -72,6 +72,7 @@ tests:
     platform_exclude:
       - m2gl025_miv
       - adp_xc7k/ae350
+      - nrf9280pdk/nrf9280/cpuppr
```

Comment:
ugh, it's still running it, does this need to be `nrf9280pdk/nrf9280/cpuppr/xip`?